### DISCUSSION TITLE
Ventola Raspberry Pi4 - Tablet consigliato

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1399] [Hardware & Firmware]
Ciao a tutti, sto facendo finalmente la migrazione su rasp.
Ho montato il tutto, ma la ventola fa un rumore esagerato, funziona perfettamente, non ha vibrazioni particolari, ma credo che la velocità sia troppa.
C'è un modo per rallentarla? 

Passando poi ad un tablet da montare a muro, i miei vecchi sono davvero obsoleti, cosa mi consigliate, qualcosa di stabile, che deve fare solo quel lavoro. Grazie

**Comment by KC-inDomus** [comment]
[leggiti questo](https://indomus.it/progetti/gestire-dinamicamente-una-ventola-tradizionale-su-raspberry-pi-via-home-assistant/).

sul tablet, uno qualunque che supporti l'app Home Assistant Companion, quindi anche uno abbastanza vecchio dovrebbe bastare e avanzare.

**Reply by snakuzzo** [reply]
...considera che se proprio vuoi, non hai nemmeno bisogno dell'app companion. Ti basta un browser

**Reply by nromagna** [reply]
Ciao per la ventola io ho preso e ne sono soddisfatto questa: Noctua NF-A4X10 5V. C'è da forare un attimono la cover ma nulla di impossibile (e se ci sono riuscito io credi è davvero facile...).
Per il tablet pensaci un attimo, a MIO PERSONALE avviso credo che "doversi" alzare per andare al tablet appeso al muro spegnere la luce ha poco senso... tutti abbiamo un cellulare praticamente sempre in tasca e se impostato bene diventa super facile per TUTTI i membri della famiglia.
Poi se il tuo è un discorso "estetico perchè fa figo" allora certo qualsiasi tabelt (diciamo degli ultimi 10 anni) può andar bene specialmente se esegui solo HA.

**Reply by KC-inDomus** [reply]
> ...considera che se proprio vuoi, non hai nemmeno bisogno dell'app companion. Ti basta un browser

ti perdi troppe funzioni del componente mobile, però.

**Reply by snakuzzo** [reply]
> > ...considera che se proprio vuoi, non hai nemmeno bisogno dell'app companion. Ti basta un browser
> 
> ti perdi troppe funzioni del componente mobile, però.

...ovvio che dipende sempre da quello che gli occorre. Dovessi farlo per me, mi basterebbe la webgui. Gli unici vantaggi che vedo per la companion sono "localiizzazione GPS", "widget", "notifiche"... che nel caso di un tablet appeso al muro trovo superflue
